### DISCUSSION TITLE
[Website Bug] Setup from Scratch SQL Typescript Postgres webpage is crashing

### DISCUSSION THREAD

**Discussion by ncphillips** [prisma/prisma#3356] [Q&A] [answered]
https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-sql-typescript-postgres

I recently tried to follow this guide and it just totally breaks. It actually crashes the browser tab.

Here's a look at the network tab:

<img width="1302" alt="image" src="https://user-images.githubusercontent.com/824015/90684790-27657900-e23f-11ea-8c1b-eb9381d1a9a7.png">

**Comment by nikolasburk** [comment, marked as answer]
Thanks for sharing this @ncphillips 🙏 We're looking into it, I created an [issue](https://github.com/prisma/docs/issues/774) in the `docs` repo and brought it up with our web development team.

**Reply by ncphillips** [reply]
Awesome! Id id manage to find a URL that worked. I got Prisma working with Next on Vercel, DB hosted in Heroku. It was pretty slick. Really enjoying the experimental migrations feature BTW!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
